Polymorphism(Java)/Method_Overloading.java

```diff
@@ -0,0 +1,39 @@
+class MethodOverloading 
+{
+       // this method accepts int
+    private static void display(int a)
+    {
+        System.out.println("Got Integer data.");
+    }
+    // this method accepts String object
+    private static void display(String a)
+    {
+        System.out.println("Got String object.");
+    }
+    //this method accepts float value
+     private static void display(float a)
```

Comment:
Incorrect indentation.
Please self the PR once.

Reply:
Like I am sorry but I can't figure out what is 4 space indentation?

Reply:
In line 14, there are 5 spaces.